static/app/views/performance/transactionSummary/transactionEvents/content.tsx

```diff
@@ -0,0 +1,242 @@
+    return (action: Actions, value: React.ReactText) => {
+      const {eventView, location} = this.props;
+
+      const searchConditions = tokenizeSearch(eventView.query);
+
+      // remove any event.type queries since it is implied to apply to only transactions
+      searchConditions.removeTag('event.type');
+
+      // no need to include transaction as its already in the query params
+      searchConditions.removeTag('transaction');
+
+      updateQuery(searchConditions, action, column, value);
+
+      browserHistory.push({
+        pathname: location.pathname,
+        query: {
+          ...location.query,
+          cursor: undefined,
+          query: stringifyQueryObject(searchConditions),
+        },
+      });
+    };
+  };
+
+  handleIncompatibleQuery: React.ComponentProps<
+    typeof CreateAlertFromViewButton
+  >['onIncompatibleQuery'] = (incompatibleAlertNoticeFn, _errors) => {
+    const incompatibleAlertNotice = incompatibleAlertNoticeFn(() =>
+      this.setState({incompatibleAlertNotice: null})
+    );
+    this.setState({incompatibleAlertNotice});
+  };
+
+  render() {
+    const {
+      eventView,
+      location,
+      organization,
+      projects,
+      transactionName,
+      limit,
+      cursorName,
+    } = this.props;
+    const {incompatibleAlertNotice} = this.state;
+
+    const transactionsListEventView = eventView.clone();
+
+    const transactionsListTitles = [
+      t('event id'),
+      t('user'),
+      t('operation duration'),
+      t('total duration'),
+      t('trace id'),
+      t('timestamp'),
+    ];
+    const cursor = decodeScalar(location.query?.[cursorName]);
+
+    return (
+      <Fragment>
+        <TransactionHeader
+          eventView={transactionsListEventView}
+          location={location}
+          organization={organization}
+          projects={projects}
+          transactionName={transactionName}
+          currentTab={Tab.Events}
+          hasWebVitals={
+            getCurrentLandingDisplay(location, projects, eventView).field ===
+            LandingDisplayField.FRONTEND_PAGELOAD
+          }
+          handleIncompatibleQuery={this.handleIncompatibleQuery}
+        />
+        <Layout.Body>
+          <StyledSdkUpdatesAlert />
+          {incompatibleAlertNotice && (
+            <Layout.Main fullWidth>{incompatibleAlertNotice}</Layout.Main>
+          )}
+          <Layout.Main fullWidth>
+            <Search {...this.props} />
+            <StyledTable>
+              <DiscoverQuery
+                location={location}
+                eventView={transactionsListEventView}
+                orgSlug={organization.slug}
+                limit={limit}
+                cursor={cursor}
+                referrer="api.discover.transactions-list"
+              >
+                {({isLoading, pageLinks, tableData}) => {
+                  return (
+                    <React.Fragment>
+                      <TransactionsTable
```

Comment:
I don't believe the `TransactionsTable` supports column resizing or re-ordering right now. Is this something we should consider?

Reply:
Hey Tony, this is something we will be addressing in this upcoming ticket:
https://getsentry.atlassian.net/browse/VIS-1079